native/ex_vl_convert/src/lib.rs

```diff
@@ -0,0 +1,250 @@
+use vl_convert_rs::converter::VgOpts;
+use vl_convert_rs::converter::VlOpts;
+use vl_convert_rs::VlConverter;
+use vl_convert_rs::VlVersion;
+
+// +-------------------------------------+
+// |            Vega Functions           |
+// +-------------------------------------+
+
+#[rustler::nif(schedule = "DirtyCpu")]
+fn vega_to_svg(vega_spec: String) -> Result<String, String> {
+    let vg_spec: serde_json::Value = match serde_json::from_str(vega_spec.as_str()) {
+        Ok(spec) => spec,
+        Err(_err) => return Err("Vega spec is not valid JSON".to_string()),
+    };
```

Comment:
We can also share this part across all functions:

```rust
fn parse_spec(spec: String) -> Result<serde_json::Value, String> {
    return match serde_json::from_str(spec.as_str()) {
        Ok(spec) => Ok(spec),
        Err(_err) => Err("The given spec is not valid JSON".to_string()),
    };
}
```

And then we do:

```rust
let vg_spec = parse_spec(vega_spec)?;
```

I think a generic error for both Vega and VegaLite is fine, because we don't expect the JSON to be invalid, since we are the ones encoding it.

Reply:
Sounds good to me 👍